### PR TITLE
Use a helper for ~related~ other posts

### LINKED ISSUE #1458: Helper for limit to 5 (related) posts
#1457

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
This introduces a helper for the (fake) related posts, instead of too much logic in the template itself. 

Close #1458

### CODE REVIEW

File: helpers/posts.rb [hunk 1]
```diff
@@ -27,4 +27,21 @@ def capitalize(tagname)
     tagname = tagname.slice(0, 1).capitalize + tagname.slice(1..-1)
     tagname.gsub(/-[a-z]/, &:upcase)
   end
+
+  def related_posts
+    case blog_controller.name
+    when :jobs
+      path = "jobs/job-preview"
```
Comment (sn3p) [inline helpers/posts.rb]: Think you can do this:

```rb
path = case blog_controller.name
  when :jobs
     "jobs/job-preview"
  when :blog
     "blog/blog-article-preview"
  end
```

small optimization :)

**Comment by RonaldDijkstra** [comment]
Name update because it actually is 5 other posts, not related. 

Maybe someday use https://github.com/dsedivec/middleman-related-articles

File: helpers/posts.rb [hunk 2]
```diff
@@ -27,4 +27,25 @@ def capitalize(tagname)
     tagname = tagname.slice(0, 1).capitalize + tagname.slice(1..-1)
     tagname.gsub(/-[a-z]/, &:upcase)
   end
+
+  # Define which partial should be used
+  def article_preview
```
Comment (sn3p) [inline helpers/posts.rb]:
```suggestion
  def article_preview_partial_path
```

would describe the function better :)

Comment (RonaldDijkstra) [reply helpers/posts.rb]: Done !

File: helpers/posts.rb [hunk 3]
```diff
@@ -27,4 +27,25 @@ def capitalize(tagname)
...
+      "blog/blog-article-preview"
+    end
+  end
+
+  # Get the other_posts
+  def other_posts
+    html = +""
+    (blog().articles - [current_article])[0...5].each do |article|
+      html << "<li>"
+      html << partial(article_preview_partial_path, locals: { article: article })
```
[the remark this answers is not in the document]
Comment (RonaldDijkstra) [reply helpers/posts.rb]: Haha acceptable.

**Pull request merged**